### DISCUSSION TITLE
ubuntu 22.04 error

### DISCUSSION THREAD

**Discussion by MohMehrnia** [ravendb/ravendb#14826] [General]
Hi, when i run ravendb 5.3.2 on ubuntu 22.04.1 show me below message

**No usable version of libssl was found**

How do resolved? (I used OpenSSL 3.0.2)

**Comment by garayx** [comment]
what is the full exception + stack trace?

**Comment by ayende** [comment]
Also, how did you install RavenDB?

**Reply by MohMehrnia** [reply]
I download and extract ravendb from the web site and run ./run.sh

when executing ./run.sh show me this error
**No usable version of libssl was found**

**Reply by ayende** [reply]
What is the _full_ error?
Is this a clean machine?

**Reply by MohMehrnia** [reply]
yes clean machine
![Untitled](https://user-images.githubusercontent.com/7066574/187193196-2462ec42-3408-41e8-9bee-cb85b529d3fc.png)

**Comment by ppekrol** [comment]
Can you try installing latest RavenDB? 5.3.104 or 5.4.1, instead of 5.3.2.

**Reply by MohMehrnia** [reply]
i need 5.3.2!!!

**Reply by gregolsky** [reply]
That's a version released prior to release of Ubuntu 22.04. There's a high chance dotnet runtime used under the hood was not ready for OpenSSL 3.0 back then. You can try 5.3.104 which is the latest on the 5.3 release channel.

**Reply by ppekrol** [reply]
5.3.2 is based on .NET 5 which is no longer supported by MS and also .NET 5 does not support Ubuntu 22.04. More info here: https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions
